config/more_samples-glyf_colr_1.py

```diff
@@ -46,40 +46,66 @@ def _cpal(color_str, alpha=1.0):
     return (_PALETTE[color], alpha)
 
 
-def _sample_sweep(accessor):
-    glyph_name = "sweep"
+def _sample_sweep(start_angle, end_angle, extend_mode_arg, color_line_range, accessor):
 
-    pen = TTGlyphPen(None)
-    pen.moveTo((100, 500))
-    pen.qCurveTo((500, 1000), (900, 500))
-    pen.qCurveTo((500, 0), (100, 500))
-    pen.closePath()
+    extend_mode_map = {
+        "reflect": ot.ExtendMode.REFLECT,
+        "repeat": ot.ExtendMode.REPEAT,
+        "pad": ot.ExtendMode.PAD,
+    }
+
+    if extend_mode_arg not in extend_mode_map:
+        return None
+
+    extend_mode = extend_mode_map[extend_mode_arg]
+
+    color_line_range_map = {
+        "narrow": {
+            "ColorStop": [
+                (0.25, *_cpal("red")),
+                (1 / 3 * 0.5 + 0.25, *_cpal("yellow")),
+                (2 / 3 * 0.5 + 0.25, *_cpal("green")),
+                (0.75, *_cpal("blue")),
+            ]
+        },
+        "wide": {
```

Comment:
"wide" seems unused?

Reply:
Removed the wide entry and added comment that this is preparation for testing with different ColorLines.